mainwindow.cpp

```diff
@@ -50,11 +50,12 @@ MainWindow::MainWindow(QWidget *parent) :
     terminal(new QProcess),
     serial(new QSerialPort),
     repeatSendTimer(new QTimer),
+    timer_serial(new QTimer),
```

Comment:
Using the timer timeout signal to trigger USB list refreshing, every foreach the /dev/xxx dir at 1s interval. I think that is not good method to solved the dynamic scanning serial devices. Wonder if it is a UNIX system signal as epoll/select asy (asynchronous notification)which is a good choice for dynamic scanning serial devices.